tabular/src/autogluon/tabular/models/mitra/sklearn_interface.py

```diff
@@ -274,6 +274,8 @@ def __init__(self,
             fine_tune_steps=DEFAULT_FINE_TUNE_STEPS,
             metric=DEFAULT_CLS_METRIC,
             state_dict=None,
+            hf_general_model=DEFAULT_GENERAL_MODEL,
+            hf_cls_model=DEFAULT_CLS_MODEL,
```

Comment:
If we are going to avoid passing `hf_reg_model`, we shouldn't even have `hf_cls_model`, and we would rename `hf_general_model` to `hf_model`? Same idea for Regressor.